erpnext/patches/v12_0/make_custom_fields_for_bank_remittance.py

```diff
@@ -3,6 +3,8 @@
 from erpnext.regional.india.setup import make_custom_fields
 
 def execute():
+    reload_doc("accounts", "doctype", "tax_category")
```

Comment:
`frappe.reload_doc()` and not `reload_doc()`

same goes for the next line as well